app/src/main/java/ar/com/wolox/android/features/homepage/news/RecyclerAdapter.kt

```diff
@@ -0,0 +1,41 @@
+package ar.com.wolox.android.features.homepage.news
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.RadioButton
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import ar.com.wolox.android.R
+import ar.com.wolox.android.models.News
+
+class RecyclerAdapter(private val dataSet: Array<News>) : RecyclerView.Adapter<RecyclerAdapter.ViewHolder>() {
```

Comment:
I suggest to move the Adapter to another file and leave this as the NewsViewHolder.
Also, you should implement the method `populate` of the ViewHolder